pkgs/tools/graphics/argyllcms/default.nix

```
@@ -15,7 +53,45 @@ stdenv.mkDerivation rec {
     hash = "sha256-t2dvbYFHEz9IUYpcM5HqDju4ugHrD7seG3QxumspxDg=";
   };
 
-  nativeBuildInputs = [ jam unzip ];
+  nativeBuildInputs = [
+    jam
+    unzip
+  ];
+
+  buildInputs =
+    [
+      libtiff
+      libjpeg
+      libpng
+      openssl
+      libusb1
+    ]
+    ++ (lib.optionals stdenv.isLinux [
```

nit: If I'm not mistaken the parenthesis can be omitted:
```suggestion
    ++ lib.optionals stdenv.isLinux [
```